documents/final-submission-document/appendices/personal-reports.tex

```diff
@@ -10,13 +10,13 @@ \subsection{Fabian Hauser}
 I find this quote very fitting for our thesis - not only was it our task to decide on options,
 but also keep questioning them to find the best possible solution.
 
-A tool we used for this were Architectural Design Decisions.
-They supported our path to find good solutions to our problems greatly in my opinion.
-The most surprising turn resulting from an architectural decision during the project was our decision to write an Angular application,
+We used Architectural Design Decisions to find and question possible solutions.
+This supported our problem solving process greatly in my opinion.
+The most surprising turn resulting from an architectural decision during the project was our decision to write a client only application,
 which I wouldn't have expected from the task description.
 
 During the project, I often had the feeling that we didn't advance as fast as I had hoped for.
-I think the main reason for this were practical limitations, as well as complexity and interdependencies of the underlying XMPP standards.
+I think the main reasons for this are practical limitations, as well as the high complexity and interdependency of underlying XMPP standards.
```

Comment:
again, `this` is not 100% clear here - eg. like `the main reason for this feeling/feel`